payment-manager/src/test/java/com/qpay/paymentmanager/service/impl/PaymentServiceImplTest.java

```diff
@@ -97,7 +97,7 @@ void should_makePaymentBetweenWallets() {
         given(walletService.getWalletById(toId)).willReturn(toWallet);
         given(walletRepository.save(expectedFromWallet)).willReturn(expectedFromWallet);
         given(walletRepository.save(expectedToWallet)).willReturn(expectedToWallet);
-        willDoNothing().given(executorService).execute(any());
+        willDoNothing().given(paymentNotificationProducer).publishPaymentNotification(any());
```

Comment:
In general it's a bad practice to use `any()`. You should validate that exact data passed to this method is correct.